{"uid":"8ac30691cc43e3d0","name":"test_check_card_price","fullName":"tests.test_main_page.TestHotSellerSection#test_check_card_price","historyId":"f4026d9f96acc9996f781ecba7818e56","time":{"start":1723253394383,"stop":1723253395337,"duration":954},"description":"This test checks that the card has a price and a price in USD","descriptionHtml":"<p>This test checks that the card has a price and a price in USD</p>\n","status":"skipped","statusMessage":"XFAIL In CI the test failed in PR#203\n\nselenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome=127.0.6533.88); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x55b0e0a9771a <unknown>\n#1 0x55b0e0768640 <unknown>\n#2 0x55b0e076e4cf <unknown>\n#3 0x55b0e07700f5 <unknown>\n#4 0x55b0e077017c <unknown>\n#5 0x55b0e07b2162 <unknown>\n#6 0x55b0e07da8e2 <unknown>\n#7 0x55b0e07ac5d9 <unknown>\n#8 0x55b0e07daaae <unknown>\n#9 0x55b0e07f908a <unknown>\n#10 0x55b0e07da683 <unknown>\n#11 0x55b0e07aad71 <unknown>\n#12 0x55b0e07ab7de <unknown>\n#13 0x55b0e0a5f2ab <unknown>\n#14 0x55b0e0a63242 <unknown>\n#15 0x55b0e0a4c665 <unknown>\n#16 0x55b0e0a63dd2 <unknown>\n#17 0x55b0e0a312af <unknown>\n#18 0x55b0e0a86eb8 <unknown>\n#19 0x55b0e0a87090 <unknown>\n#20 0x55b0e0a964ec <unknown>\n#21 0x7f9b2d494ac3 <unknown>","statusTrace":"self = <test_main_page.TestMainPage.TestHotSellerSection object at 0x7f3f30eaec10>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b9fa57032f949958316e6c304ef92298\")>\n\n    @pytest.mark.xfail(reason=\"In CI the test failed in PR#203\")\n    def test_check_card_price(self, driver):\n        \"\"\"This test checks that the card has a price and a price in USD\"\"\"\n        page = MainPage(driver, MAIN_PAGE_URL)\n        page.open()\n>       text = page.check_card_price()\n\ntests/test_main_page.py:48: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/main_page.py:49: in check_card_price\n    text = self.get_text(self.locators.CARD_PRICE)\npages/base_page.py:145: in get_text\n    return text.text\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:89: in text\n    return self._execute(Command.GET_ELEMENT_TEXT)[\"value\"]\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:394: in _execute\n    return self._parent.execute(command, params)\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f3f31729910>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"stale element reference\",\"message\":\"stale element reference: stale elemen...\\\\n#19 0x55b0e0a87090 \\\\u003Cunknown>\\\\n#20 0x55b0e0a964ec \\\\u003Cunknown>\\\\n#21 0x7f9b2d494ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\nE         (Session info: chrome=127.0.6533.88); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nE       Stacktrace:\nE       #0 0x55b0e0a9771a <unknown>\nE       #1 0x55b0e0768640 <unknown>\nE       #2 0x55b0e076e4cf <unknown>\nE       #3 0x55b0e07700f5 <unknown>\nE       #4 0x55b0e077017c <unknown>\nE       #5 0x55b0e07b2162 <unknown>\nE       #6 0x55b0e07da8e2 <unknown>\nE       #7 0x55b0e07ac5d9 <unknown>\nE       #8 0x55b0e07daaae <unknown>\nE       #9 0x55b0e07f908a <unknown>\nE       #10 0x55b0e07da683 <unknown>\nE       #11 0x55b0e07aad71 <unknown>\nE       #12 0x55b0e07ab7de <unknown>\nE       #13 0x55b0e0a5f2ab <unknown>\nE       #14 0x55b0e0a63242 <unknown>\nE       #15 0x55b0e0a4c665 <unknown>\nE       #16 0x55b0e0a63dd2 <unknown>\nE       #17 0x55b0e0a312af <unknown>\nE       #18 0x55b0e0a86eb8 <unknown>\nE       #19 0x55b0e0a87090 <unknown>\nE       #20 0x55b0e0a964ec <unknown>\nE       #21 0x7f9b2d494ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: StaleElementReferenceException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1723252925247,"stop":1723252925311,"duration":64},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder","time":{"start":1723252925311,"stop":1723252925311,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1723253393137,"stop":1723253394383,"duration":1246},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"This test checks that the card has a price and a price in USD","status":"skipped","statusMessage":"XFAIL In CI the test failed in PR#203\n\nselenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome=127.0.6533.88); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x55b0e0a9771a <unknown>\n#1 0x55b0e0768640 <unknown>\n#2 0x55b0e076e4cf <unknown>\n#3 0x55b0e07700f5 <unknown>\n#4 0x55b0e077017c <unknown>\n#5 0x55b0e07b2162 <unknown>\n#6 0x55b0e07da8e2 <unknown>\n#7 0x55b0e07ac5d9 <unknown>\n#8 0x55b0e07daaae <unknown>\n#9 0x55b0e07f908a <unknown>\n#10 0x55b0e07da683 <unknown>\n#11 0x55b0e07aad71 <unknown>\n#12 0x55b0e07ab7de <unknown>\n#13 0x55b0e0a5f2ab <unknown>\n#14 0x55b0e0a63242 <unknown>\n#15 0x55b0e0a4c665 <unknown>\n#16 0x55b0e0a63dd2 <unknown>\n#17 0x55b0e0a312af <unknown>\n#18 0x55b0e0a86eb8 <unknown>\n#19 0x55b0e0a87090 <unknown>\n#20 0x55b0e0a964ec <unknown>\n#21 0x7f9b2d494ac3 <unknown>","statusTrace":"self = <test_main_page.TestMainPage.TestHotSellerSection object at 0x7f3f30eaec10>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b9fa57032f949958316e6c304ef92298\")>\n\n    @pytest.mark.xfail(reason=\"In CI the test failed in PR#203\")\n    def test_check_card_price(self, driver):\n        \"\"\"This test checks that the card has a price and a price in USD\"\"\"\n        page = MainPage(driver, MAIN_PAGE_URL)\n        page.open()\n>       text = page.check_card_price()\n\ntests/test_main_page.py:48: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/main_page.py:49: in check_card_price\n    text = self.get_text(self.locators.CARD_PRICE)\npages/base_page.py:145: in get_text\n    return text.text\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:89: in text\n    return self._execute(Command.GET_ELEMENT_TEXT)[\"value\"]\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:394: in _execute\n    return self._parent.execute(command, params)\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f3f31729910>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"stale element reference\",\"message\":\"stale element reference: stale elemen...\\\\n#19 0x55b0e0a87090 \\\\u003Cunknown>\\\\n#20 0x55b0e0a964ec \\\\u003Cunknown>\\\\n#21 0x7f9b2d494ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\nE         (Session info: chrome=127.0.6533.88); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nE       Stacktrace:\nE       #0 0x55b0e0a9771a <unknown>\nE       #1 0x55b0e0768640 <unknown>\nE       #2 0x55b0e076e4cf <unknown>\nE       #3 0x55b0e07700f5 <unknown>\nE       #4 0x55b0e077017c <unknown>\nE       #5 0x55b0e07b2162 <unknown>\nE       #6 0x55b0e07da8e2 <unknown>\nE       #7 0x55b0e07ac5d9 <unknown>\nE       #8 0x55b0e07daaae <unknown>\nE       #9 0x55b0e07f908a <unknown>\nE       #10 0x55b0e07da683 <unknown>\nE       #11 0x55b0e07aad71 <unknown>\nE       #12 0x55b0e07ab7de <unknown>\nE       #13 0x55b0e0a5f2ab <unknown>\nE       #14 0x55b0e0a63242 <unknown>\nE       #15 0x55b0e0a4c665 <unknown>\nE       #16 0x55b0e0a63dd2 <unknown>\nE       #17 0x55b0e0a312af <unknown>\nE       #18 0x55b0e0a86eb8 <unknown>\nE       #19 0x55b0e0a87090 <unknown>\nE       #20 0x55b0e0a964ec <unknown>\nE       #21 0x7f9b2d494ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: StaleElementReferenceException","steps":[{"name":"Open a browser","time":{"start":1723253394383,"stop":1723253395207,"duration":824},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Get element text","time":{"start":1723253395208,"stop":1723253395336,"duration":128},"status":"broken","statusMessage":"selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome=127.0.6533.88); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x55b0e0a9771a <unknown>\n#1 0x55b0e0768640 <unknown>\n#2 0x55b0e076e4cf <unknown>\n#3 0x55b0e07700f5 <unknown>\n#4 0x55b0e077017c <unknown>\n#5 0x55b0e07b2162 <unknown>\n#6 0x55b0e07da8e2 <unknown>\n#7 0x55b0e07ac5d9 <unknown>\n#8 0x55b0e07daaae <unknown>\n#9 0x55b0e07f908a <unknown>\n#10 0x55b0e07da683 <unknown>\n#11 0x55b0e07aad71 <unknown>\n#12 0x55b0e07ab7de <unknown>\n#13 0x55b0e0a5f2ab <unknown>\n#14 0x55b0e0a63242 <unknown>\n#15 0x55b0e0a4c665 <unknown>\n#16 0x55b0e0a63dd2 <unknown>\n#17 0x55b0e0a312af <unknown>\n#18 0x55b0e0a86eb8 <unknown>\n#19 0x55b0e0a87090 <unknown>\n#20 0x55b0e0a964ec <unknown>\n#21 0x7f9b2d494ac3 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 145, in get_text\n    return text.text\n           ^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py\", line 89, in text\n    return self._execute(Command.GET_ELEMENT_TEXT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py\", line 394, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Find a visible element","time":{"start":1723253395208,"stop":1723253395289,"duration":81},"status":"passed","steps":[{"name":"Find a present element","time":{"start":1723253395208,"stop":1723253395227,"duration":19},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', \"li[class='product-item']:nth-child(2) span[class='price']\")"},{"name":"timeout","value":"5"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Go to specified element","time":{"start":1723253395227,"stop":1723253395240,"duration":13},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"element","value":"<selenium.webdriver.remote.webelement.WebElement (session=\"b9fa57032f949958316e6c304ef92298\", element=\"f.1B4092EBBD293B3212AB859C2E2E9E37.d.213E8EB4A393B5ED0BEADAE12A18B08D.e.10\")>"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', \"li[class='product-item']:nth-child(2) span[class='price']\")"},{"name":"timeout","value":"10"}],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', \"li[class='product-item']:nth-child(2) span[class='price']\")"}],"shouldDisplayMessage":true,"stepsCount":3,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"13a96b1173fea6eb","name":"Screenshot","source":"13a96b1173fea6eb.png","type":"image/png","size":133658},{"uid":"2114d4dcd2b2139f","name":"HTML source","source":"2114d4dcd2b2139f.html","type":"text/html","size":123028}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":5,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1723253395549,"stop":1723253395619,"duration":70},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder::0","time":{"start":1723254142198,"stop":1723254142198,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"epic","value":"Main Page"},{"name":"feature","value":"Testing Hot Seller Section"},{"name":"tag","value":"@pytest.mark.xfail(reason='In CI the test failed in PR#203')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_main_page"},{"name":"subSuite","value":"TestMainPage"},{"name":"host","value":"fv-az1432-390"},{"name":"thread","value":"2874-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_main_page"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":1,"passed":6,"unknown":0,"total":7},"items":[{"uid":"4acae4d29aa7f5df","reportUrl":"https://verafes.github.io/luma_project/397//#testresult/4acae4d29aa7f5df","status":"passed","statusDetails":"XPASS In CI the test failed in PR#203","time":{"start":1723167097012,"stop":1723167098047,"duration":1035}},{"uid":"955bfd2c800b2223","reportUrl":"https://verafes.github.io/luma_project/396//#testresult/955bfd2c800b2223","status":"passed","statusDetails":"XPASS In CI the test failed in PR#203","time":{"start":1723080539346,"stop":1723080540242,"duration":896}},{"uid":"91de25428c851718","reportUrl":"https://verafes.github.io/luma_project/395//#testresult/91de25428c851718","status":"passed","statusDetails":"XPASS In CI the test failed in PR#203","time":{"start":1722994171089,"stop":1722994172058,"duration":969}},{"uid":"d19f8e6979636ee8","reportUrl":"https://verafes.github.io/luma_project/394//#testresult/d19f8e6979636ee8","status":"passed","statusDetails":"XPASS In CI the test failed in PR#203","time":{"start":1722907723532,"stop":1722907724535,"duration":1003}},{"uid":"b65ee99785ee30b5","reportUrl":"https://verafes.github.io/luma_project/393//#testresult/b65ee99785ee30b5","status":"passed","statusDetails":"XPASS In CI the test failed in PR#203","time":{"start":1722821366305,"stop":1722821367167,"duration":862}},{"uid":"e8c00b752d91dc36","reportUrl":"https://verafes.github.io/luma_project/392//#testresult/e8c00b752d91dc36","status":"passed","statusDetails":"XPASS In CI the test failed in PR#203","time":{"start":1722735353764,"stop":1722735355116,"duration":1352}}]},"tags":["@pytest.mark.xfail(reason='In CI the test failed in PR#203')"]},"source":"8ac30691cc43e3d0.json","parameterValues":[]}